lua/nvim-ts-autotag/internal.lua

```diff
@@ -546,6 +546,10 @@ M.attach = function(bufnr, lang)
                     vim.api.nvim_buf_set_text(bufnr or 0, row - 1, col, row - 1, col, { '>' })
                     M.close_tag()
                     vim.api.nvim_win_set_cursor(0, { row, col + 1 })
+                    local has_luasnip, luasnip = pcall(require, 'luasnip')
```

Comment:
Will this not potentially break some setups? Requiring 'unrelated' plugins feels wrong.

Reply:
I could be wrong but I think it is fine because of the `require` call being wrapped in the `pcall` method. If the user doesn't have luasnip then `has_luasnip` evaluates to `false` and nothing happens. I tested and this also seems to not incur any overhead unless the user does have luasnip and the module has not yet been loaded.

Reply:
function `require `will scan all runtime path to find luasnip . use it like that may not be the best approach

 `close_tag` function is public. you can turn of enable_close and write your own mapping.